source/MulensModel/tests/test_BinaryLens.py

```diff
@@ -21,6 +21,46 @@ def test_small_q():
     np.testing.assert_almost_equal(result, 3.6868957, decimal=3)
 
 
+def test_vbbl_vs_point_source():
+    """
+    check if vbbl and point source calculations give the same result
```

Comment:
Add something to the documentation specifying that this is a Shear test.

Reply:
Good point. I'll move it to `test_BinaryLensWithShear.py`

Reply:
Done.